ltrace.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: ltrace
+  version: 0.7.3
+  epoch: 0
+  description: Tracks runtime library calls in dynamically linked programs
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - autoconf
+      - automake
+      - elfutils-dev
+      - libtool
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 0e6f8c077471b544c06def7192d983861ad2f8688dd5504beae62f0c5f5b9503
+      uri: https://www.ltrace.org/ltrace_${{package.version}}.orig.tar.bz2
+
+  - uses: patch
+    with:
+      # https://git.alpinelinux.org/aports/plain/main/ltrace/mips.patch
+      patches: mips.patch
+
+  - uses: patch
+    with:
+      # https://git.alpinelinux.org/aports/plain/main/ltrace/musl.patch
+      patches: musl.patch
+
+  - uses: patch
+    with:
+      # https://git.alpinelinux.org/aports/plain/main/ltrace/ltrace_aarch64.patch
+      patches: ltrace_aarch64.patch
```

Comment:
Why do we need these?

Reply:
they were available in APKBUILD

Reply:
See explanation here -> https://github.com/wolfi-dev/os/pull/3637#discussion_r1264675856

esp.:
>Wolfi's reason for including a patch should not be "because another distro did it"

Reply:
I confirmed that the build was succeed without these patches, so, I removed them 🟢